crates/astria-bridge-withdrawer/src/bridge_withdrawer/submitter/mod.rs

```diff
@@ -87,7 +87,8 @@ pub(super) struct Submitter {
     signer: SequencerKey,
     sequencer_chain_id: String,
     startup_tx: oneshot::Sender<SequencerStartupInfo>,
-    expected_fee_asset_id: asset::Id,
+    expected_fee_asset: asset::Denom,
+    expected_fee_asset_ibc: asset::denom::IbcPrefixed,
```

Comment:
aren't these are the same (ibc-prefixed is just a hash of the denom)? why do we need both here?

Reply:
You are right: it's enough to construct the `_ibc` one once in the setup method (so that it's not reconstructed in the setup's 2 loops):  3a8066b